testbed/tests/scenarios.go

```diff
@@ -658,9 +663,8 @@ func getLogsID(logToRetry []plog.Logs) []string {
 		logRecord := logElement.ResourceLogs().At(0).ScopeLogs().At(0).LogRecords()
 		for index := 0; index < logRecord.Len(); index++ {
 			logObj := logRecord.At(index)
-			itemIndex, _ := logObj.Attributes().Get("item_index")
```

Comment:
Out of curiosity, is this change required for the test that this PR adds? From what I can see it should be fine to include it but just want to understand why.